### PR TITLE
fix golint issues

### LINKED ISSUE #90: Fix golint issues
At the moment, golint $(go list ./...) reports quite a few issues (missing  comments and stutters).
We should run golint with -set_exit_status flag after they are fixed - see #85

### PR DESCRIPTION
**Pull request by marpio**
Fixes #90

### CODE REVIEW

File: actions/storage.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/spf13/afero"
 )
 
-func newStorage() (storage.Storage, error) {
+func newStorage() (storage.Backend, error) {
```
Comment (michalpristas) [inline actions/storage.go]: minor: i understand that linter will complain about package and interface name being the same, but backend does not make sense to me. Client, GetterSaver, Provider non of them are good tbh but i don;t like backend for some reason

Comment (marpio) [reply actions/storage.go]: I wasn't sure myself. Service maybe? but it's very general...

Comment (michalpristas) [reply actions/storage.go]: what about driver? makes sense?

Comment (arschles) [reply actions/storage.go]: Could we move this decision to a follow-up?

**Comment by michalpristas** [comment]
naming was a minor thing, not blocker

**Pull request merged**